packages/flutter_tools/gradle/deprecated_settings.gradle

```diff
@@ -0,0 +1,31 @@
+include ':app'
+
```

Comment:
This file is used to check for the existing `settings.gradle` files. This file has two variants separated by `;EOF`. I can't add a comment since that would change the pattern that it checks for in `gradle.dart`.